python/pyproject.toml

```diff
@@ -1,18 +1,28 @@
-import logging
-from setuptools import find_packages, setup
-from src.aac import __version__
-from os import linesep
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
 
-README_FILE_PATH = "../README.md"
-
-try:
-    with open(README_FILE_PATH, "r") as file_handler:
-        readme_description = file_handler.read()
-except Exception as exception:
-    logging.error(f"Failed to open readme file {README_FILE_PATH} with error:{linesep}  {exception}")
-    readme_description = ""
+[project]
+name = "aac"
+version =  "0.4.18"
+authors = [
+    { name="bunchw", email="asdfasdaf@email.com" },
+    { name="lizzcondrey", email="asdfasdfd@mail.com" },
```

Comment:
Remember we were wanting to try and shift this to being the team label instead so that we don't have to keep adding team members names.